great-idea/css/index.css

```diff
@@ -152,7 +152,8 @@ h5 {
 /* Copy and paste your work from yesterday here and start to refactor into flexbox */
 
 .container {
-  margin: 5%;
+  margin: 5% auto;
+  max-width: 700px;
```

Comment:
Hi Darragh, since the image width is 880px, try setting your max-width to that size at well. It should resolve the banner image overflowing the container.